File: app/src/main/java/com/tammamkhalaf/myuaeguide/categories/hereDeveloper/DiscoverExploreResponse.java

package com.tammamkhalaf.myuaeguide.categories.hereDeveloper;

import com.google.gson.annotations.Expose;
import com.google.gson.annotations.SerializedName;

public class DiscoverExploreResponse {

    @SerializedName("search")
    @Expose
    private Search search;
    @SerializedName("results")
    @Expose
    private Results results;

    public Search getSearch() {
        return search;
    }

    public void setSearch(Search search) {
        this.search = search;
    }

    public Results getResults() {
        return results;
    }

    public void setResults(Results results) {
        this.results = results;
    }

}

File: shared/src/commonMain/kotlin/com/tammamkhalaf/kmmsharedmodule/categories/hereDeveloper/discoverExplore/DiscoverExploreResponse.kt
package com.tammamkhalaf.myuaeguide.categories.hereDeveloper.discoverExplore

import com.google.gson.annotations.Expose

class DiscoverExploreResponse {
    @SerializedName("results")
    @Expose
    private var results: com.tammamkhalaf.myuaeguide.categories.hereDeveloper.discoverExplore.Results? = null

    @SerializedName("search")
    @Expose
    private var search: com.tammamkhalaf.myuaeguide.categories.hereDeveloper.discoverExplore.Search? = null
    fun getResults(): com.tammamkhalaf.myuaeguide.categories.hereDeveloper.discoverExplore.Results? {
        return results
    }

    fun setResults(results: com.tammamkhalaf.myuaeguide.categories.hereDeveloper.discoverExplore.Results?) {
        this.results = results
    }

    fun getSearch(): com.tammamkhalaf.myuaeguide.categories.hereDeveloper.discoverExplore.Search? {
        return search
    }

    fun setSearch(search: com.tammamkhalaf.myuaeguide.categories.hereDeveloper.discoverExplore.Search?) {
        this.search = search
    }
}